### DISCUSSION TITLE
Training in Azure ML Studio with custom data

### DISCUSSION THREAD

**Discussion by NatPwrAA** [ultralytics/yolov5#7934] [Q&A]
I've got a custom dataset that I'd like to use to train a model in Azure ML Studio. 

Found this : https://docs.microsoft.com/en-us/azure/machine-learning/tutorial-auto-train-image-models

Is that the only way to do it? I wanted to check if anybody else has used this to train and if they could track the metrics whilst training?

Update:  The issue I'm faced with right now is that I can't seem to upload the labels as .txt as some of them are empty with no annotations in them.

**Comment by glenn-jocher** [comment]
@NatPwrAA I haven't use it myself but it looks like any other python environment, i.e. Google Colab, so you should be able to run the same code you run in Colab with the same workflows.

**Reply by NatPwrAA** [reply]
Thanks for the reply. I'll keep this discussion updated with any issues I have and solutions I find, might help someone in the future.
